src/main/java/org/galatea/starter/service/IexClient.java

```diff
@@ -32,4 +34,7 @@
   @GetMapping("/tops/last")
   List<IexLastTradedPrice> getLastTradedPriceForSymbols(@RequestParam("symbols") String[] symbols);
 
+  @GetMapping("/stable/stock/{symbol}/chart/{range}/{date}?chartByDay=true")
```

Comment:
Why do we want the chart by day = true for all requests

Reply:
When making the request with a specific date in mind, adding this to the end of the query will return only the OHLCV chart information for that date.